spring-context/src/main/java/org/springframework/scheduling/config/ScheduledTaskRegistrar.java

```diff
@@ -122,6 +122,14 @@ public void setTriggerTasks(Map<Runnable, Trigger> triggerTasks) {
 	public void setTriggerTasksList(List<TriggerTask> triggerTasks) {
 		this.triggerTasks = triggerTasks;
 	}
+	
+	/**
+	 * Return trigger tasks as a list of {@link TriggerTask}
+	 * @since 4.2
+	 */
+	public List<TriggerTask> getTriggerTaskList() {
+		return this.triggerTasks;
```

Comment:
Regardless of whether we want to do this or not, we shouldn't expose the raw mutable list. This also means that your tests must be rewritten. 

Before doing that, let's discuss on the Jira issue what we want to do here. Thanks!